.github/workflows/release.yaml

```diff
@@ -3,12 +3,52 @@ on:
   push:
     tags:
       - "v*"
+  workflow_dispatch: {}
+  schedule:
+    - cron: "45 0 * * *"
 
 env:
   CARGO_TERM_COLOR: always
   RUSTFLAGS: "-D warnings"
+  VERSION: "${{ github.event_name == 'push' && github.ref_name || 'nightly' }}"
+
+permissions:
+  contents: write
+  id-token: write
```

Comment:
Why are these two permissions here? I don't feel very comfortable giving a workflow commit or OIDC permissions.

Reply:
`id-token` was mistakenly added from the SBOM PR. I'll remove it.
`contents: write` is necessary to change release assets. It was already given before, just not explicit.